azureLogIngestion/adapter/utils/snakecase.ts

```diff
@@ -0,0 +1,14 @@
+import * as _ from "lodash"
```

Comment:
Importing * from lodash can increase bundle size quite a bit. I'd think we could do these lodash methods as separate imports.